src/tiled/invertYCoordinateHelper.h

```diff
@@ -0,0 +1,19 @@
+#ifndef INVERTYCOORDINATEHELPER_H
+#define INVERTYCOORDINATEHELPER_H
```

Comment:
Please use this instead of include guards:
```c++
#pragma once
```

Reply:
Thanks for the pointers. i apologize for all the silly mistakes, i'll get this cleaned up as soon as i can.